docs/contributors/development.md

```diff
@@ -6,7 +6,7 @@
 
 ## Requirements
 
-- `Node.js` 18.x (may work with other versions but without any guarantee). For [nvm](https://github.com/nvm-sh/nvm) users, just run `nvm use` for the repository root
+- `Node.js`: 20.x (may work with other versions but without any guarantee). For [nvm](https://github.com/nvm-sh/nvm) users, just run `nvm use` for the repository root
 - `npm` 8 (may work with other versions but without any guarantee). In particular, we use [lockfileVersion: 2](https://docs.npmjs.com/cli/v8/configuring-npm/package-lock-json#lockfileversion). To ensure you use the (latest) npm 8 version, run `npm i -g npm@8`
```

Comment:
**thought:** npm version is probably no longer accurate
We could also stop referencing the versions here and just say, use the node version define in `.nvmrc` and the npm version associated to the node version

Reply:
```suggestion
- `Node.js`: use the version referenced in `.nvmrc` file (may work with other versions but without any guarantee). For [nvm](https://github.com/nvm-sh/nvm) users, just run `nvm use` for the repository root
- `npm`: use the version associated to the node version.
```